skrub/_minhash_encoder.py

```diff
@@ -244,6 +248,11 @@ def fit(self, X, y=None) -> "MinHashEncoder":
         :obj:`MinHashEncoder`
             The fitted :class:`MinHashEncoder` instance (self).
         """
+        self._check_n_features(X, reset=True)
+        self._check_feature_names(X, reset=True)
+        X = check_input(X)
+        self.n_features_in_ = X.shape[1]
```

Comment:
This line can be removed since `self.n_features_in_` is already set in `self._check_n_features`. WDYT?